### DISCUSSION TITLE
Is there compatibility issues with the Dreambooth extension?

### DISCUSSION THREAD

**Discussion by FugueSegue** [vladmandic/sdnext#1811] [Q&A] [answered]
I am constantly having trouble creating models in the Dreambooth extension. It's not an error with training itself. It's the creating of the models to prepare for training, the first step of the process.

I tried deleting the entire SD.Next installation and reinstalling everything. It worked once. Then it might work for a while. And then after a reboot, I tried it again and I got the same error.

I tried installing Automatic1111's webui and the Dreambooth extension in that. No problem. Reboot the computer, tried again. No problem.

I'd post the errors I got but I think that would be pointless since they all clearly pertain to the Dreambooth extension.

I just want to know if there are known issues with SD.Next and compatibility with the Dreambooth extension?

**Comment by vladmandic** [comment, marked as answer]
yes. dreambooth extension install much older requirements than ones used by sdnext and causes quite a few different issues. this goes back for more than 6 months and it was never resolved.
it also causes problems in a1111, they are just less noticeable.

author has stated several times he does not plan to improve extension anymore and his goal is to have a separate standalone tool for dreambooth training.

**Reply by FugueSegue** [reply]
Thanks for the response, Vlad.

I learned something new just now. I don't know if it is helpful.

TL;DR: The "default settings" are somehow related to the problem.

When I installed Automatic1111's webui today along with the Dreambooth extension, I did not make any changes to the settings. I had no trouble creating Dreambooth models. I shut down A1111, rebooted, restarted A1111. No problem creating models.

That got me thinking. I was wondering if there was a setting that I usually apply that is causing the problem. Whenever I reinstall SD.Next (or A1111's webui when I used it) the first thing I do is go to the settings tab and make about a dozen changes, apply the changes, and then restart. Whenever I do this with SD.Next, I notice a tremendous amount of changes being display at the top of the tab. I don't recognize any of the changes that are displayed.

I tried an experiment. I deleted SD.Next again, reinstalled it, installed Dreambooth, shut down SD.Next, and restarted it so that Dreambooth could finish installation. Without making any changes to the settings, I went straight to the Dreambooth tab. I created a model without any problem. I tried it again, and again. No problem. I shut down SD.Next, rebooted, launched SD.Next. Again, I had no problem creating a model.

Now I went to the settings tab. Without making any changes to the settings at all, I clicked "Apply settings". I got a wall of 53 setting changes displayed at the top of the tab. Similar to what I experienced in the past with SD.Next installation. I shut down and restarted SD.Next, and went back to the Dreambooth tab. I tried to create a model and it failed in the exact same way it's been doing for days.

I tried to "Restore defaults" in the settings and restart SD.Next. Same story, Dreambooth was broken. Back in the settings, I clicked "Apply settings" and got the same 53 changes displays.

I have no idea if any of my observations are helpful. Perhaps it might shed some light on a related issue.

**Reply by vladmandic** [reply]
They are helpful, but unless dreambooth changes it's behavior not to modify system requirements, I will not look it at.
